mtags/src/main/scala/scala/meta/internal/metals/JdkSources.scala

```diff
@@ -42,7 +44,7 @@ object JdkSources {
 
     for {
       javaHomeString <- userJavaHome.orElse(defaultJavaHome).toList
-      javaHome = AbsolutePath(javaHomeString)
+      javaHome <- Try(AbsolutePath(javaHomeString.trim())).toOption.toList
```

Comment:
But in case of https://github.com/scalameta/metals/issues/5230 it seems like `javaHomeString` was resolved to ` `, so it seems like trimming won't exactly give you what you want. Or am I missing something?

Reply:
Oh but I think it is printed `Trailing char < > at index 0:  `, see you have two spaces one for formatting and the other is the path (at least I think that's the case, if you search the web the exception prints the erroneous path).